### PR TITLE
DOCSUP-11552: Fixed and updated documentation

### PR DESCRIPTION
**Pull request by gyuton**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #25596

### CODE REVIEW

**Comment by gyuton** [comment]
@Mergifyio refresh

File: docs/ru/sql-reference/functions/tuple-map-functions.md [hunk 1]
```diff
@@ -73,22 +73,22 @@ SELECT a['key2'] FROM table_map;
...
+Аргументами являются контейнеры [Map](../../sql-reference/data-types/map.md) или [кортежи](../../sql-reference/data-types/tuple.md#tuplet1-t2) из двух [массивов](../../sql-reference/data-types/array.md#data-type-array), где элементы в первом массиве представляют ключи, а второй массив содержит значения для каждого ключа.
 Все массивы ключей должны иметь один и тот же тип, а все массивы значений должны содержать элементы, которые можно приводить к одному типу ([Int64](../../sql-reference/data-types/int-uint.md#int-ranges), [UInt64](../../sql-reference/data-types/int-uint.md#uint-ranges) или [Float64](../../sql-reference/data-types/float.md#float32-float64)).
 Общий приведенный тип используется в качестве типа для результирующего массива.
 
 **Возвращаемое значение**
 
--   Возвращает один [кортеж](../../sql-reference/data-types/tuple.md#tuplet1-t2), в котором первый массив содержит отсортированные ключи, а второй — значения.
+-   В зависимости от аргументов возвращает один [Map](../../sql-reference/data-types/map.md) или [кортеж](../../sql-reference/data-types/tuple.md#tuplet1-t2), в котором первый массив содержит отсортированные ключи, а второй — значения.
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/tuple-map-functions.md:87]:
```suggestion
-   В зависимости от типа аргументов возвращает один [Map](../../sql-reference/data-types/map.md) или [кортеж](../../sql-reference/data-types/tuple.md#tuplet1-t2), в котором первый массив содержит отсортированные ключи, а второй — значения.
```

File: docs/ru/sql-reference/functions/tuple-map-functions.md [hunk 2]
```diff
@@ -102,6 +102,20 @@ SELECT mapAdd(([toUInt8(1), 2], [1, 1]), ([toUInt8(1), 2], [1, 1])) as res, toTy
 └───────────────┴────────────────────────────────────┘
 ```
 
+Запрос с типом `Map`:
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/tuple-map-functions.md:105]:
```suggestion
Запрос с контейнером `Map`:
```

**Review by evillique — APPROVED** [review]

**Comment by gyuton** [comment]
@Mergifyio update

File: docs/en/sql-reference/functions/tuple-map-functions.md [hunk 3]
```diff
@@ -78,15 +78,15 @@ mapAdd(arg1, arg2 [, ...])
...
+Arguments are [maps](../../sql-reference/data-types/map.md) or [tuples](../../sql-reference/data-types/tuple.md#tuplet1-t2) of two [arrays](../../sql-reference/data-types/array.md#data-type-array), where items in the first array represent keys, and the second array contains values for the each key. All key arrays should have same type, and all value arrays should contain items which are promoted to the one type ([Int64](../../sql-reference/data-types/int-uint.md#int-ranges), [UInt64](../../sql-reference/data-types/int-uint.md#uint-ranges) or [Float64](../../sql-reference/data-types/float.md#float32-float64)). The common promoted type is used as a type for the result array.
 
 **Returned value**
 
 -   Depending on the arguments returns one [map](../../sql-reference/data-types/map.md) or [tuple](../../sql-reference/data-types/tuple.md#tuplet1-t2), where the first array contains the sorted keys and the second array contains values.
 
 **Example**
 
-Query with a tuple map:
+Query with a Tuple:
```
Comment (evillique) [inline docs/en/sql-reference/functions/tuple-map-functions.md:89]:
```suggestion
Query with a tuple:
```

**Review by evillique — APPROVED** [review]

**Pull request merged**